local-test-configuration/monitoring/index.html

```diff
@@ -858,6 +865,23 @@ <h2>Active data feeds</h2>
       return sponsorWallet;
     };
 
+    const deriveSponsorWallet = (
+      sponsorWalletMnemonic,
+      dapiNameOrDataFeedId,
+      updateParameters,
+      walletDerivationScheme
+    ) => {
+      // Derive the hash of the data feed using which the sponsor wallet is derived. For self-funded feeds it's more
+      // suitable to derive the hash also from update parameters. This does not apply to mananaged feeds which want to be
+      // funded by the same wallet independently of the update parameters.
+      const sponsorAddressHash =
```

Comment:
This is the sponsor address, not the sponsor address hash, no?

Reply:
No, I've separated the hashing part into separate functions and one base function (`deriveSponsorWalletFromSponsorAddressHash`) which takes first 20 bytes from the hash and interprets that as sponsor address and derives the wallet.

Reply:
I mean the thing that you split into chunks and build the path out of is called the sponsor address, regardless if it's the address of an actual sponsor (such as [these](https://github.com/api3dao/operations-database/blob/48676e0e79114b5250f21cf7518100b0eb5e6645/src/scripts/contants.ts))

Reply:
Yeah, that's why I don't want to name it `sponsorAddress`. There are:
- `sponsorAddressHash` - Hash from which we get the build path - e.g. `ethers.keccak256(ethers.solidityPacked(['bytes32', 'bytes'], [dapiNameOrDataFeedId, updateParameters]))`
- `sponsorAddress` - `ethers.getAddress(sponsorAddressHash.slice(0, 42))`
- `sponsorWallet` - Derived from mnemonic and `sponsorAddress`

Not sure if we are not misunderstanding each other. LMK if I am missing something.

Reply:
Yeah okay, I hadn't dug deep enough to find `sponsorAddress`